### ISSUE TITLE
[nx-remotecache-s3] v0.2.1 dependencies

### ISSUE THREAD

**Issue by hoonoh** [robinpellegrims/pellegrims#88]
With v0.2.1 update, dependencies seem to be dropped. Is this intended?
```
 >  NX   There was a critical error when running your command

   Cannot find module 'nx-remotecache-custom'
Require stack:
- (...)/node_modules/@pellegrims/nx-remotecache-s3/src/lib/nx-remotecache-s3.js
- (...)/node_modules/@pellegrims/nx-remotecache-s3/src/index.js
- (...)/node_modules/nx/src/tasks-runner/run-command.js
- (...)/node_modules/nx/src/command-line/affected.js
- (...)/node_modules/nx/src/command-line/nx-commands.js
- (...)/node_modules/nx/bin/init-local.js
- (...)/node_modules/nx/bin/nx.js
```

This can also be seen in npm:
https://www.npmjs.com/package/@pellegrims/nx-remotecache-s3/v/0.2.1
0 dependencies
https://www.npmjs.com/package/@pellegrims/nx-remotecache-s3/v/0.2.0
3 dependencies: ([nx-remotecache-custom](https://www.npmjs.com/package/nx-remotecache-custom), [@aws-sdk/client-s3](https://www.npmjs.com/package/%40aws-sdk%2Fclient-s3), [get-stream](https://www.npmjs.com/package/get-stream))

**Comment by robinpellegrims** [comment]
Fixed and released in [v0.2.2](https://github.com/robinpellegrims/pellegrims/releases/tag/nx-remotecache-s3-0.2.2)

**Comment by robinpellegrims** [comment]
After some further investigation I found out that there's a [flag](https://nx.dev/packages/js/executors/tsc#buildableprojectdepsinpackagejsontype) in the new [@nrwl/js:tsc](https://nx.dev/packages/js/executors/tsc#nrwljstsc) executor to specify if buildable lib dependencies should be automatically put in the package's dependencies or peer dependencies. The default value is peer dependencies, which explains the original cause of this issue.

I refactored the previous fix into using the flag so that dependencies/versions are added automatically. Released in [v0.2.3](https://github.com/robinpellegrims/pellegrims/releases/tag/nx-remotecache-s3-0.2.3).
